src/data/past-clients.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "logoSrc": "../images/client-logos/walden.svg",
+    "name": "Walden Local"
```

Comment:
should we add in the imageAlt here as well?

Reply:
Oh I see what you meant about the imageAlt.

I did some more reading about good image alts and I think we should do something descriptive that interpolates with the client name, something like `Logo of past client company, ${name}`. I think for this I'd keep the data source as just name and add that interpolation in on the view side